src/containers/EmployeeList/Employee/styles.module.scss

```diff
@@ -0,0 +1,151 @@
+.container {
+  padding: 30px 3%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.contentHeader {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 30px;
+}
+
+.mainBox {
+  position: relative;
+  box-shadow: 0 0.1rem 0.25rem rgb(0 0 0 / 8%);
+  display: flex;
+  flex-direction: column;
+  min-width: 0;
+  word-wrap: break-word;
+  background-color: #fff;
+  background-clip: border-box;
+  border: 0 solid rgba(222, 226, 230, 70%);
+  border-radius: 0.25rem;
+}
+
+.contentTitle {
+  margin: 0;
+}
+
+.boxHeader {
+  padding: 1.3rem;
+  margin-bottom: 0;
+  background-color: #fff;
+  border-bottom: 1px solid rgba(222, 226, 230, 70%);
+}
+
+.containerBoxHeader {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: calc(0 * -1);
+  margin-right: calc(1.5rem / -2);
+  margin-left: calc(1.5rem / -2);
+}
+
+.searchBox {
+  flex: 0 0 auto;
+  width: 50%;
+}
+
+.selectionBox {
+  padding-right: calc(1.5rem / 2);
+  padding-left: calc(1.5rem / 2);
+  margin-top: 0;
+  flex: 0 0 auto;
+  width: 25%;
+}
+
+@media (max-width: 770px) {
+  .searchBox {
+    width: 100%;
+  }
+
+  .selectionBox {
+    width: 50%;
+  }
+}
+
+.boxBody {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.boxBodyContainer {
+  padding: 1.3rem;
+}
+
+.userBoxContainer {
+  width: 50%;
+  padding: 0.5rem;
+}
+@media (max-width: 500px) {
+  .userBoxContainer {
+    width: 100%;
+  }
+}
+
+@media (min-width: 1000px) {
+  .userBoxContainer {
+    width: 25%;
+  }
+}
```

Comment:
In case the media query has only one selector. Consider adding that media query to corresponding selector.

For example: 

```scss
.userBoxContainer {
   width: 50%;
   padding: 0.5rem;

   @media (max-width: 500px) {
       width: 100%;
   }
 }
```